shared/actions/chat/creators.js

```diff
@@ -98,8 +99,9 @@ function updateLatestMessage (conversationIDKey: Constants.ConversationIDKey): C
   return {payload: {conversationIDKey}, type: 'chat:updateLatestMessage'}
 }
 
-function badgeAppForChat (conversations: List<Constants.ConversationBadgeState>): Constants.BadgeAppForChat {
-  return {payload: conversations, type: 'chat:badgeAppForChat'}
+function badgeAppForChat (conversations: ?Array<RPCTypes.BadgeConversationInfo>): Constants.BadgeAppForChat {
```

Comment:
moved this translation logic in here vs in the notifications-listeners, it knew too much about chat